packages/web-runtime/package.json

```diff
@@ -13,7 +13,7 @@
     "lodash-es": "^4.17.21",
     "luxon": "^1.27.0",
     "oidc-client": "1.11.5",
-    "owncloud-design-system": "^9.3.0",
+    "owncloud-design-system": "10.0.0-rc2",
```

Comment:
Needs to be `10.0.0-rc1`, the rc2 was mistakenly released and doesn't exist on npmjs.